services-manager/pom.xml

```diff
@@ -42,6 +42,10 @@
         <tag>HEAD</tag>
     </scm>
 
+    <properties>
+        <examples.app.version>0.5.0-${revision}</examples.app.version>
```

Comment:
whats this property used for?

Reply:
This is used in the profile yaml files, specifically the example-libs profile, when adding the example-library jar to the service classpaths.